src/interactive/azext_interactive/azclishell/scenario_suggest.py

```diff
@@ -32,12 +44,43 @@ def update(self, sample: str):
                 self.param_value_map[cur_param] += ' ' + part
                 self.param_value_map[cur_param] = self.param_value_map[cur_param].strip()
 
-    def get_suggestion(self, cli, buffer, document):
+
+        def update_customized_cached_param_map(self, command_text: str):
+        """Update the value of a parameter in the customized parameter value map"""
+        # remove the 'az ' part of command
+        command_text = command_text.split('az ')[-1]
+        cur_param = ''
```

Comment:
Personally, I think it would be more accurate to name it as `cached_param`
```suggestion
        cached_param = ''
```

Reply:
Received, already modified in https://github.com/wu5bocheng/azure-cli-extensions/pull/2/commits/f92b35511e194d8a6630747c490e364c43b2039a